README.md

```diff
@@ -187,7 +192,7 @@ The `"Two" | "Active" | "Beside"` are popular values from `ViewColumn` to specif
 
 The session watcher allows RStudio addins to be executed via an `{rstudioapi}` emulation layer.
 
-To enable this feature, set `options(vsc.rstudioapi = TRUE)` in your .Rprofile. 
```

Comment:
Didn't know about this when I edited the README.md. This was done automatically by VSCode with my settings.json `editor.trimAutoWhitespace = true`